src/analyze/analyze.c

```diff
@@ -1249,6 +1367,47 @@ static int dot(int argc, char *argv[], void *userdata) {
 
         printf("digraph systemd {\n");
 
+        /* user configuration below ~/.config */
+        r = sd_path_home(SD_PATH_USER_CONFIGURATION, "", &user_config_dir);
+        if (r < 0)
+                return r;
+
+        r = strv_extend(&config_dirs, "/etc");
+        if (r < 0)
+                return log_oom();
+
+        r = strv_extend(&config_dirs, "/run");
+        if (r < 0)
+                return log_oom();
+
+        r = strv_extend(&config_dirs, "/usr/local/lib");
+        if (r < 0)
+                return log_oom();
+
+        r = strv_extend(&config_dirs, "/usr/lib");
+        if (r < 0)
+                return log_oom();
+
+#if HAVE_SPLIT_USR
+        r = strv_extend(&config_dirs, "/lib");
+        if (r < 0)
+                return log_oom();
+#endif
+
+        r = strv_extend(&config_dirs, user_config_dir);
+        if (r < 0)
+                return log_oom();
```

Comment:
please split all this out to a helper function of its own. use src/tmpfiles/tmpfiles.c as inspiration

Reply:
done